Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -173,7 +176,7 @@ class TopTabsViewController: UIViewController {
     }
     
     func privateTabTapped() {
-        delegate?.topTabsDidPressPrivateTab()
+        delegate?.topTabsDidPressPrivateTab(isPrivate ? lastNormalTab : lastPrivateTab)
```

Comment:
Yeah, true, I think I need to rename this to make more sense.  Would topTabsDidTogglePrivateMode make more sense? :)